ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/CVEsTable.tsx

```diff
@@ -211,6 +217,19 @@ function CVEsTable({
                                             }
                                         />
                                     </Td>
+                                    {isNvdCvssEnabled && (
+                                        <Td dataLabel="Top NVD CVSS">
+                                            <CvssFormatted
+                                                cvss={0.0}
+                                                // scoreVersion={
+                                                //     scoreVersions.length > 0
+                                                //         ? scoreVersions.join('/')
+                                                //         : undefined
+                                                // }
```

Comment:
You included this commented-out logic for calculating the score version but not in the ones below. Is that intentional?

Reply:
Good eyes. Adapted helper functions in a subsequent commit.